cmd/create/accountroles/cmd.go

```diff
@@ -307,30 +304,11 @@ func buildCommands(prefix string, permissionsBoundary string, accountID string,
 			name, aws.GetPolicyARN(accountID, policyName))
 		commands = append(commands, createRole, createPolicy, attachRolePolicy)
 	}
-
-	for credrequest, operator := range credRequests {
-		name := aws.GetPolicyName(prefix, operator.Namespace(), operator.Name())
-		iamTags := fmt.Sprintf(
-			"Key=%s,Value=%s Key=%s,Value=%s Key=%s,Value=%s Key=%s,Value=%s",
-			tags.OpenShiftVersion, defaultPolicyVersion,
-			tags.RolePrefix, prefix,
-			"operator_namespace", operator.Namespace(),
-			"operator_name", operator.Name(),
-		)
-		createPolicy := fmt.Sprintf("aws iam create-policy \\\n"+
-			"\t--policy-name %s \\\n"+
-			"\t--policy-document file://openshift_%s_policy.json \\\n"+
-			"\t--tags %s",
-			name, credrequest, iamTags)
-		commands = append(commands, createPolicy)
-	}
```

Comment:
This block might need to be added to the `buildCommands` of the `create operator-roles` command.

Reply:
Ah, I see that this already exists there. Nevemrind.